tools/sponge/preproc_scripts/fill_glorys_nn_monthly.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/tcsh
```

Comment:
very minor, but if these are cshell scripts then maybe we should change the file extension to .csh

Reply:
@theresa-morrison , since you're using C shell syntax (e.g., set), it might be simpler to rename the script to `*.csh` to reflect that.

Reply:
done!